dandi/tests/test_dandiarchive.py

```diff
@@ -441,7 +441,7 @@ def test_parse_dandi_url_unknown_instance() -> None:
         parse_dandi_url("dandi://not-an-instance/000001")
     assert str(excinfo.value) == (
         "Unknown instance 'not-an-instance'.  Valid instances: dandi,"
-        " dandi-api-local-docker-tests, dandi-staging"
+        " dandi-api-local-docker-tests, dandi-staging, linc, linc-staging"
```

Comment:
might be worth right away to compose this from `known_instances` to avoid hardcoding. WDYT?

Reply:
This is just the unit test case -- in the prod. code it isn't hard-coding

Did you have in-mind the hard-coding being removed in the unit test here as well?

Reply:
yeap.  I see no reason to hardcode them in the test here.  The purpose is to test to see that all are listed in the particular order.  Code could be used to provide that and avoid future edits happen we need to add one more instance

Reply:
on this note, chime in to
- https://github.com/dandi/dandi-cli/issues/1521
- https://github.com/dandi/dandi-cli/issues/1522

Reply:
> yeap. I see no reason to hardcode them in the test here. The purpose is to test to see that all are listed in the particular order. Code could be used to provide that and avoid future edits happen we need to add one more instance

Updated the test here